www/index.html

```diff
@@ -906,6 +917,41 @@
           </div>
         {{/each}}
       </script>
+
+      <script type="text/template7" id="galleryTemplate">
+        {{#each album}}
+          <a href="album/{{id}}">
+            <div class="albums">
+              <div class="album-full-background" style="background-image: url({{thumb}})"></div>
+              <div class="album-text">
+                <div class="title"><p> {{title}} </p></div>
+                <div><i class="fa fa-calendar"></i> {{js "new Date(this.start_date).shortDateString()"}}</div>
```

Comment:
You can add this to the loop in `applyTemplate` in the js file instead which will clean up the template a bit.